src/services/categories.services.ts

```diff
@@ -1,17 +1,21 @@
 import {api} from "@/boot/axios";
+import CategoryItem from "@/models/category/categories";
 
 export default new class {
-    private readonly category = 'categories';
-    getCategory(id: number){
-        return api.get(this.category + "/" + id.toString())
+    private readonly endpoint = 'categories';
+    getCategory(id: string){
```

Comment:
Cái này là convention thôi, có thể lúc làm dự án này nọ sẽ bắt buộc làm như thế này.

Nhưng anh suggest là viết ngắn nhất có thể (tất nhiên phải đảm bảo không bị nhập nhằng)
```suggestion
    get(id: string){
```

Reply:
Thêm dòng trống vào từng member

Reply:
done